disabled/nodesCoordinatorCache.go

```diff
@@ -1,6 +1,6 @@
 package disabled
 
-// NodesCoordinatorCache is a disabled implementation of Cacher interface
+// NodesCoordinatorCache is a disabled implementation of nodesCoordinator interface
```

Comment:
the interface being implemented is actually Cacher

I think also you should rename NodesCoordinatorCache here to Cache, as this could be used also in other contexts not only for nodes coordinator.

Reply:
done